cmake/linux/CMakeLists.txt

```diff
@@ -2,3 +2,16 @@ INSTALL(FILES lmms.png                  DESTINATION "${DATA_DIR}/pixmaps")
 INSTALL(FILES lmms                      DESTINATION "${DATA_DIR}/menu")
 INSTALL(FILES lmms.desktop              DESTINATION "${DATA_DIR}/applications")
 INSTALL(FILES lmms.xml                  DESTINATION "${DATA_DIR}/mime/packages")
+
+IF(LMMS_HOST_X86)
+	SET(HOST_ARCH i386-linux-gnu)
+ELSEIF(LMMS_HOST_X86_64)
+	SET(HOST_ARCH x86_64-linux-gnu)
+ELSE()
+	SET(HOST_ARCH unknown-linux-gnu)
+ENDIF()
+SET(RUN_FILE "${CMAKE_PROJECT_NAME}-${VERSION}-${HOST_ARCH}.run")
+ADD_CUSTOM_TARGET(run-installer
```

Comment:
Semantics, but In this case, it's a packager, not an installer, right?  We could even call the target `makeself`.